Formula/n/nerdfetch.rb

```diff
@@ -0,0 +1,16 @@
+class Nerdfetch < Formula
+  desc "POSIX *nix (Linux, macOS, Android, *BSD, etc) fetch script using Nerdfonts"
+  homepage "https://github.com/ThatOneCalculator/NerdFetch"
+  url "https://github.com/ThatOneCalculator/NerdFetch/archive/refs/tags/v8.1.2.tar.gz"
+  sha256 "aa7c63d190963dee9fa1bc68e426d56d965d544b0d1ce0b38a55d26278d71dd2"
+  license "MIT"
+  head "https://github.com/ThatOneCalculator/NerdFetch.git", branch: "main"
+
+  def install
+    bin.install "nerdfetch"
+  end
+
+  test do
+    system bin/"nerdfetch"
```

Comment:
maybe `assert_match "brew"` just to test the output?

Reply:
yeah, that works for me, I just adopt the other *fetch tests :)